### PR TITLE
Homepage Information Icon

### PR DESCRIPTION
**Pull request by mab6bf**
- Adds an information icon to the homepage
- Adds a tooltip to the information icon that activates on hover or tap
  - Tooltip can be closed through clicking/tapping an X button at the top right of the tooltip
- Fixes the Header bar CSS for Firefox

Issue: https://federal-spending-transparency.atlassian.net/browse/DEV-553

- [x] Code Review
- [x] Design Review

### CODE REVIEW

**Comment by gitapablabah** [comment]
Looking great! A couple comments...
-- Lower icon height, make closer to headline and period. See mock for visual guidance: https://projects.invisionapp.com/d/main/#/console/8586929/282110527/preview
-- If the 'i' icon is a transparency of #FFFFFF, then make a little brighter for 508 compliance. Sorry I didn't call that out before. Feel free to ping me the percentage and I will get you an exact number.
-- Add a colon after Data Source:

**Comment by mab6bf** [comment]
Changes made. I used the `lighten(#205493, 90%)` from the style guide for the icon color. Let me know if you are looking for another tone.

**Comment by gitapablabah** [comment]
All edits look good. TY!

File: src/js/components/homepage/hero/Hero.jsx [hunk 1]
```diff
@@ -5,27 +5,71 @@
 
 import React from 'react';
 
+import * as Icons from 'components/sharedComponents/icons/Icons';
```
[the remark this answers is not in the document]
Comment (mab6bf) [reply src/js/components/homepage/hero/Hero.jsx]: Yep

File: src/js/components/homepage/hero/HeroTooltip.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        const windowWidth = window.innerWidth;
+        if ((iconLeft + tooltipWidth) > windowWidth) {
+            iconLeft = windowWidth - tooltipWidth - margin;
+        }
+
+        return { iconTop, iconLeft };
+    }
+
+    render() {
+        const position = this.getPosition();
```
[the remark this answers is not in the document]
Comment (mab6bf) [reply src/js/components/homepage/hero/HeroTooltip.jsx]: Yep

**Pull request merged**